idaes/surrogate/pysmo/sampling.py

```diff
@@ -314,6 +314,100 @@ def data_sequencing(self, no_samples, prime_base):
         sequence_decimal = sequence_decimal.reshape(sequence_decimal.shape[0], )
         return sequence_decimal
 
+    def selection_columns_preprocessing(self, data_input, xlabels, ylabels):
+        """
+        Pre-processing data for multiple output selection case.
+        
+        Args:
+            data_input:     data supplied by user (dataframe or numpy array)
+            xlabels:        list of input variables
+            ylabels:        list of output variables
+        """
+        self.df_flag = True
+
+        if isinstance(data_input, pd.DataFrame):
+            if xlabels is None:
+                xlabels = []
+            if ylabels is None:
+                ylabels = []
+            set_of_labels = xlabels + ylabels
+
+            if not set_of_labels: 
+                # Empty list assumes one output in final column - all other variables treated as inputs
+                print('No column information provided. All except last column will be considered as x variables.')
```

Comment:
this should be a log statement

Reply:
Changing all the print statements in PySMO to logs will an time-consuming undertaking and will be addressed in a separate PR later.